### PR TITLE
Updated crosswalk awsx example.

### PR DESCRIPTION
**Pull request by tusharshahrs**
### Proposed changes

1. Fixed aws vpc crosswalk arguments
2. Fixed the security group name, since (sg-) does not work as per amazon.
3. Added a section of How to use vpc, security group, and ec2 instance

### Related issues (optional)
 Fix for [4765](https://github.com/pulumi/docs/issues/4765)
<!--Refer to related PRs or issues: #1234, or 'Fixes #1234' or 'Closes #1234'.

### CODE REVIEW

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 1]
```diff
@@ -474,6 +474,84 @@ sg.createEgressRule("outbound-access", {
...
+
+const server = new aws.ec2.Instance("webserver-www", {
+    instanceType: size,
+    vpcSecurityGroupIds: [ sg.id ], // reference the security group resource above
+    subnetId: pulumi.output(vpc.publicSubnetIds)[0],  // reference the public subnet from the custom vpc above
+    ami: ami.id,
+});
+```
+
+This example launches an ec2 instance in the vpc and security group that was created above.
```
Comment (clstokes) [inline content/docs/guides/crosswalk/aws/vpc.md:511]:
```suggestion
If we run `pulumi up`, the `aws.ec2.Instance` will be provisioned using the _first_ public subnet from the `awsx.ec2.Vpc` component and the security group provisioned with the `awsx.ec2.SecurityGroup` component:
```

File: content/docs/guides/crosswalk/aws/vpc.md [hunk 2]
```diff
@@ -474,6 +474,84 @@ sg.createEgressRule("outbound-access", {
 For additional details about configuring security group rules, See the
 [Security Groups for Your VPC](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html) documentation.
 
+## How to use your VPC, Security Group, and EC2 instance
+
```
Comment (clstokes) [inline content/docs/guides/crosswalk/aws/vpc.md:478]:
```suggestion
## How to use your VPC, Security Group, and EC2 instance

This example shows how to deploy an EC2 instance using a VPC and Security Group provisioned with the Crosswalk AWS component:

```

**Review by clstokes — CHANGES_REQUESTED** [review]
Please make sure the suggested updates render correctly and then feel free to merge.

File: content/docs/guides/crosswalk/aws/vpc.md (hunk 1 above)
Comment (tusharshahrs) [reply content/docs/guides/crosswalk/aws/vpc.md]: Updated the content to reflect the above

File: content/docs/guides/crosswalk/aws/vpc.md (hunk 2 above)
Comment (tusharshahrs) [reply content/docs/guides/crosswalk/aws/vpc.md]: Updated the content to reflect the above

**Review by pierskarsenbarg — APPROVED** [review]

**Pull request merged**